src/models/ServerStyleSheet.js

```diff
@@ -26,12 +26,18 @@ const streamBrowserErr =
     : ''
 
 export default class ServerStyleSheet {
-  closed: boolean
   instance: StyleSheet
+  closed: boolean
+  cleanup: () => void
 
   constructor() {
-    this.instance = StyleSheet.master.clone()
+    const masterSheet = StyleSheet.master
     this.closed = false
+    this.instance = masterSheet.clone()
+    this.cleanup = () => {
```

Comment:
this code is starting to get gross again :/ maybe we can put `this.closed = true` inside `cleanup` as well to DRY up a little

Reply:
that's all cleaned up now :)